fireapp/lib/presentation/shift_request/shift_request.dart

```diff
@@ -20,64 +21,92 @@ class _ShiftRequestViewState extends State<ShiftRequestView> {
-                              ],
-                            );
-                          }).toList(),
-                        ],
-                      );
-                    }).toList(),
-                  ),
-                ),
-              );
-            } else if (state is ExceptionRequestState) {
-              return Center(child: Text('Error:'));
-            }
-          }
-          return SizedBox.shrink(); // Default empty state
+      body: RequestStateWidget.stream<List<ShiftRequest>>(
+        state: _viewModel.shiftRequests,
+        retry: () => _viewModel.loadShiftRequests(widget.requestId),
+        child: (_, shiftRequests) {
+          return SingleChildScrollView(
+            child: Padding(
+              padding: const EdgeInsets.symmetric(horizontal: 16.0),
+              child: Column(
+                crossAxisAlignment: CrossAxisAlignment.start,
+                children: shiftRequests.map((shiftRequest) {
+                  return Column(
+                    children: [
+                      _buildContainer(
+                        ShiftRequestWidget(title: 'Shift ID', content: shiftRequest.shiftID),
+                      ),
+                      _buildContainer(
+                        ShiftRequestWidget(title: 'Asset Class', content: shiftRequest.assetClass),
+                      ),
+                      _buildContainer(
+                        ShiftRequestWidget(title: 'Start Time', content: shiftRequest.startTime.toIso8601String()),
+                      ),
+                      _buildContainer(
+                        ShiftRequestWidget(title: 'End Time', content: shiftRequest.endTime.toIso8601String()),
+                      ),
+                      ...shiftRequest.shiftVolunteers.map((volunteer) {
+                        return Column(
+                          children: [
+                            _buildContainer(
+                              ShiftRequestWidget(title: 'Volunteer Name', content: '${volunteer.volunteerGivenName} ${volunteer.volunteerSurname}'),
+                            ),
+                            _buildContainer(
+                              ShiftRequestWidget(title: 'Mobile Number', content: volunteer.mobileNumber),
+                            ),
+                            _buildContainer(
+                              ShiftRequestWidget(title: 'Position ID', content: volunteer.positionId.toString()),
+                            ),
+                            _buildContainer(
+                              ShiftRequestWidget(title: 'Role', content: volunteer.role),
+                            ),
+                            _buildContainer(
+                              ShiftRequestWidget(title: 'Status', content: volunteer.status),
+                            ),
+                          ],
+                        );
+                      }).toList(),
```

Comment:
Please space all the elements of this list by 0.5.rdp(). You can do this easily by calling `.spacedBy(0.5.rdp())` on the list (you can see an example of this in `reset_email.dart`.